src/include/migraphx/matcher.hpp

```diff
@@ -1041,6 +1041,14 @@ inline auto has_attribute(const std::string& name)
         [=](instruction_ref ins) { return ins->get_operator().attributes().contains(name); });
 }
 
+inline auto has_op_value(const std::string& name, const std::string& value)
+{
+    return make_basic_pred_matcher([=](instruction_ref ins) {
+        auto op_val = ins->get_operator().to_value();
+        return op_val.contains(name) and op_val[name].to<std::string>() == value;
+    });
+}
```

Comment:
Shouldn't we have a further check here to make sure that the value of the name is actually a string type? For example, in an op like squeeze, "axes" would not return a string.